l1-contracts/src/core/Rollup.sol

```diff
@@ -55,6 +61,13 @@ contract Rollup is Decoder {
 
     rollupStateHash = newStateHash;
 
+    // @todo (issue #605) handle fee collector
+    IInbox inbox = REGISTRY.getInbox();
```

Comment:
would it be more gas efficient to just make one external call getting all addresses then destructure?

Reply:
If getting all the addresses we will be reading an extra storage slot for the rollup address as well, e.g., 1 contract access + 3 storage reads. Here we have 1 cold contract access and 1 hot + 2 storage reads.